lib/views/index.erb

```diff
@@ -0,0 +1,18 @@
+<div id='new-peep'>
+	<form action="/peeps" method="post">
+		Add a peep!
```

Comment:
If you have content like this, make sure it's wrapped in a tag.

The standard for titles in forms like this is to use:

```html
<form>
  <fieldset>
    <legend>Add a Peep!</legend>
    <!-- all your inputs -->
  </fieldset>
</form>
```